<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视觉算法参数调整</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>正极过渡片</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>二值化参数</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetT1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetT2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>周长范围</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetCmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetCmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>面积范围</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetSmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pSheetSmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bPsheetTrig">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>相机触发</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bSaveData">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="pSheetInfo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>正极盖板</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>二值化参数</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateT1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateT2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>周长范围</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateCmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateCmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>面积范围</string>
      </property>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateSmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="pPlateSmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bPplateTrig">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>相机触发</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bSaveData_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>负极过渡片</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>二值化参数</string>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetT1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetT2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>周长范围</string>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetCmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetCmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>面积范围</string>
      </property>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetSmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nSheetSmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bNsheetTrig">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>相机触发</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bSaveData_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>负极盖板</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>二值化参数</string>
      </property>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateT1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateT2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>周长范围</string>
      </property>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateCmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateCmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>面积范围</string>
      </property>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateSmin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="nPlateSmax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bNplateTrig">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>相机触发</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bSaveData_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存参数</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bSaveData_2</sender>
   <signal>clicked()</signal>
   <receiver>bSaveData</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSaveData_3</sender>
   <signal>clicked()</signal>
   <receiver>bSaveData</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSaveData_4</sender>
   <signal>clicked()</signal>
   <receiver>bSaveData</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
